src/vma/sock/sockinfo_tcp.h

```diff
@@ -322,6 +322,9 @@ class sockinfo_tcp : public sockinfo, public timer_handler
 	static err_t ack_recvd_lwip_cb(void *arg, struct tcp_pcb *tpcb, u16_t space);
 	static err_t rx_lwip_cb(void *arg, struct tcp_pcb *tpcb, struct pbuf *p, err_t err);
 	static err_t rx_drop_lwip_cb(void *arg, struct tcp_pcb *tpcb, struct pbuf *p, err_t err);
+        
+	// poll once CQ. Motivation: in case sndbuf size is 0 - try polling for a pending ACK in CQ
+	unsigned rx_poll_once_non_blocking(int & err);
```

Comment:
Coding style could be updated but that should be on the entire file (even the entire file system)  and not in just one line.